docs/guides/operations-guide/openstack/flavor-manager.md

```diff
@@ -4,3 +4,122 @@ sidebar_position: 51
+
+```bash
+pip3 install openstack-flavor-manager
+```
+
+Or you can clone the repository from https://github.com/osism/openstack-flavor-manager and run the installation
+from the source code:
+
+```bash
+tox -- --help
+```
+
+After installation, you maybe have to reload your shell.
+
+## Usage
+
+Usage: python -m openstack_flavor_manager.main [OPTIONS]
+
+--name               TEXT  Name of flavor definitions. [default: scs] \
+--debug                    Enable debug logging.\
+--cloud              TEXT  Cloud name in clouds.yaml. [default: admin]\
+--recommended              Create recommended flavors.\
+--help                     Show this message and exit.
+
+For example, if you want to deploy the recommended SCS flavors, you can simply run:
+
+Only install mandatory flavors:
+
+```bash
+~$ openstack-flavor-manager
+```
+
+Also install SCS-recommended flavors if required:
+
+```bash
+~$ openstack-flavor-manager --recommended
+```
+
+the Output should look like this on success:
+```
+2023-09-20 13:03:14 | INFO     | Flavor 'SCS-1V-4' created.
+2023-09-20 13:03:14 | INFO     | Flavor 'SCS-2V-8' created.
+2023-09-20 13:03:14 | INFO     | Flavor 'SCS-4V-16' created.
+2023-09-20 13:03:14 | INFO     | Flavor 'SCS-8V-32' created.
+...
+```
+
+All recommended flavors should now be available in your OpenStack cloud environment.
+Check yourself by running: 
+
+```bash
+openstack flavor list
+```
+
+The --cloud parameter is optional. If you have multiple clouds configured in your clouds.yaml, 
+you can specify the cloud to connect to using this parameter.
+By default, the "admin" cloud is used if no other is specified.
+
+The clouds.yaml file is a configuration file that contains credentials and endpoint information for connecting
+to OpenStack clouds. The openstack.connect(cloud=cloud) function reads this file to establish a connection to the 
+specified OpenStack cloud environment.
+
+
+## Flavor definitions
+
+There are two different configuration available by default.
+One for "scs" and one for "osism." Each has its own "Mandatory" and "Recommended" flavor set.
+If you run the program with a specific configuration (either "scs" or "osism"), using the --name parameter, 
+the list of flavors to install changes:
+
+e.g.:
+
+```bash
+python -m flavor_manager.main scs
+```
+installs Flavor1 and Flavor2 (from the "scs" configuration).
+
+```bash
+python -m flavor_manager.main --recommended scs
+```
+installs Flavor1, Flavor2, Flavor3, and Flavor4 (also from the "scs" configuration).
+
+```bash
+python -m flavor_manager.main osism
+```
+installs Flavor1 and Flavor5 (from the "osism" configuration).
+
+```bash
+python -m flavor_manager.main --recommended osism
+```
+installs Flavor1, Flavor5, Flavor3, and Flavor6 (from the "osism" configuration).
+
+If you first run python -m flavor_manager.main --recommended and then python -m flavor_manager.main osism, 
+only "Flavor5" will be installed.
+This is because Flavor1 is already installed, 
```

Comment:
This does not make sense without context (yaml)

Reply:
SCS and OSISM references have been added to the docs